### ISSUE TITLE
Improvements to documentation

### ISSUE THREAD

**Issue by danielengblom** [wasakredit/php-checkout-sdk#4]
Since we want integrators to use `order_references` instead of `order_reference_id` we should remove `order_reference_id` from our example for createcheckout
```
$payload = array(
  'order_reference_id' => 'a1234567890-1337',
  'order_references' => array(
    [0] => array(
      'key' => 'magento_quote_id',
      'value' => $orderId
    )
```

**Comment by lusffs** [comment]
We've also got reports that the example code of $payload on line 115 is missing an index.
```
$payload = array(
  'items' => array(
    'financed_price' => array(
      'amount' => '14995.00',
      'currency' => 'SEK'
    ),
    'product_id' => '12345'
  )
);
```

Should be corrected to the following:

```
$payload = array(
  'items' => array(
    [0] => array(
      'financed_price' => array(
        'amount' => '14995.00',
        'currency' => 'SEK'
      ),
      'product_id' => '12345'
    )
  )
);
```

**Comment by lusffs** [comment]
This issue has been resolved with PR #6
